manualdirectdebit.php

```diff
@@ -306,6 +306,9 @@ function manualdirectdebit_civicrm_buildForm($formName, &$form) {
   if ($formName === 'CRM_Member_Form_Membership' || $formName === 'CRM_Member_Form_MembershipRenewal') {
     $customGroupInjector = new CRM_ManualDirectDebit_Hook_BuildForm_InjectCustomGroup($form);
     $customGroupInjector->buildForm();
+
+    $customGroupInjector = new CRM_ManualDirectDebit_Hook_BuildForm_Membership($form);
```

Comment:
`$customGroupInjector` might not be the best name for this object. Maybe, `$formBuilder`?